config/application.rb

```diff
@@ -28,6 +28,6 @@ class Application < Rails::Application
     # config.i18n.default_locale = :de
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
-    #config.active_record.raise_in_transactional_callbacks = true
+    config.active_record.raise_in_transactional_callbacks = true
```

Comment:
I have no problem with waiting on this.  Just wanted to get it off my VM before I forgot or wiped it clean.